src/models/primer.ts

```diff
@@ -11,7 +11,7 @@ export interface IPrimer {
    * @param config Configuration object for the SDK. Use this to set listeners, theme,
    * and payment method specific settings.
    */
-  showUniversalCheckout(token: String, config: PrimerConfig): void;
+  showUniversalCheckout(token: string | null, config: PrimerConfig): void;
```

Comment:
[Question]

Same question here as before, why are we allowing the token to be null in these methods?

Reply:
As above